Assets/Scripts/Framework/TargetSelector.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Framework
+{
+    public class TargetSelector : MonoBehaviour
+    {   
+        [SerializeField] private List<Transform> targetList;
+        [SerializeField] private LineRenderer lineRenderer;
+        [SerializeField] private Transform currentTarget;
```

Comment:
```suggestion
        [SerializeField] private Transform currentPoint;
```
Zelfde als hierboven. Plus vraag me af of dit `[SerializeField]` nodig heeft.